lib/turn_message.rb

```diff
@@ -1,5 +1,12 @@
 class TurnMessage
   def self.for(result)
+    # if reasult.size == 1
```

Comment:
Try to avoid committing commented out code, by removing commented out code before committing. It tends to distract from the task at hand.